README.md

```diff
@@ -49,13 +56,28 @@ Compile-time dependencies (only when installing from source):
 Test-time dependencies:
  * `Test::Fatal`
 
-There is a small part in the code that may not be compatible with non-Unix operating systems, in that it assumes that the file /dev/null exists.
+There is a small part in the code that may not be compatible with non-Unix
+operating systems, in that it assumes that the file /dev/null exists.
 
 ## Installation and verification
 
 ### Recommended installation
 
-The recommended way to install Zonemaster::LDNS is to install it from CPAN as a dependency to Zonemaster::Engine. If you follow the [installation instructions for Zonemaster::Engine](https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Installation.md) you will get all the prerequisites and dependencies for Zonemaster::LDNS before installing it from CPAN.
+The recommended way to install Zonemaster::LDNS is to install it from CPAN as a
+dependency to Zonemaster::Engine. If you follow the
+[installation instructions for Zonemaster::Engine] you will get all the
+prerequisites and dependencies for Zonemaster::LDNS before installing it from
+CPAN.
```

Comment:
This section feels a bit dated too. We should probably recommend the Debian packaged version over CPAN.

Reply:
Yes, but it is enough to say that it should be installed as a dependency. I'll update.

Reply:
The first sentence promises apples and delivers oranges. Then the second sentence delivers the apples. I suggest striking a few words:

> The recommended way to install Zonemaster::LDNS is to ~install it as a
dependency to Zonemaster::Engine. If you~ follow the

Reply:
Fixed.